easybuild/easyconfigs/n/Nek5000/Nek5000-17.0-intel-2018a.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'PackedBinary'
+
+name = 'Nek5000'
+version = '17.0'
+
+homepage = 'https://nek5000.mcs.anl.gov/'
+description = "a fast and scalable high-order solver for computational fluid dynamics"
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/Nek5000/Nek5000/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_hypre_blas.patch']
+checksums = [
+    '4d8d4793ce3c926c54e09a5a5968fa959fe0ba46bd2e6b8043e099528ee35a60',  # v17.0.tar.gz
+    '55d66818151b63826b55545051145a0b0d806d4df7fcf5cf57cf2c75f461f441',  # Nek5000-17.0_hypre_blas.patch
+]
+
+builddependencies = [('CMake', '3.10.2')]
+
+dependencies = [('X11', '20180131')]
+
+buildininstalldir = True
+
+install_cmd = (
+    'mv %(name)s-%(version)s/{*,.[^.]*} . && rmdir %(name)s-%(version)s && cd tools && ./maketools all &&'
+    'cd ../3rd_party/gslib && env GSLIB_OPT="USREXIT=1 BLAS=2 PREFIX=gslib_ FPREFIX=fgslib_" ./install &&'
+    'cd ../cvode && ./install'
+)
+
+postinstallcmds = [
+    'sed -i -e "s|^#FC=.*|FC=$FC|" bin/makenek',
+    'sed -i -e "s|^#CC=.*|CC=$CC|" bin/makenek',
+    'sed -i -e "s|^#FFLAGS=.*|FFLAGS=\\"$FFLAGS\\"|" bin/makenek',
+    'sed -i -e "s|^#CFLAGS=.*|CFLAGS=\\"$CFLAGS\\"|" bin/makenek',
```

Comment:
@hajgato The tests that verifies whether a dumped easyconfig file is fully equivalent with the original trips over this (the escaped inner double quotes), you may need to use a workaround like use `<doublequote>` first, which you then replace at the end with `"` in an additional command...

Reply:
Why not include this in the patch file instead?

Reply:
@boegel I do not icluse this in a patch, becasue I need the value of `$FC`, `$CC`, `$FFLAG`, `$CFLAG` environmental variables.